src/components/markdown-help.yml

```diff
@@ -158,7 +158,7 @@ blockHelpMarkdown:
 
     Bitte beachte, dass Link- bzw. Bildadressen keine Leerzeichen enthalten dürfen, um korrekt dargestellt zu werden.
 
-    Darüberhinaus kannst du auf die gleiche Weise in einigen Feldern (gekennzeichnet mit der Option „Medien wiedergeben“), anstelle von Bildern auch Video- bzw. Audio-Dateien einbinden.
+    Darüberhinaus kannst du auf die gleiche Weise von Bildern auch Video- bzw. Audio-Dateien einbinden.
```

Comment:
"anstelle" is missing between "Weise" and "von"

Reply:
Re-corrected, thanks!

Reply:
Cool!